rskj-core/src/main/java/co/rsk/Start.java

```diff
@@ -27,16 +27,16 @@
  * The entrypoint for the RSK full node
  */
 public class Start {
-
     private static final Logger logger = LoggerFactory.getLogger("start");
-
     public static void main(String[] args) {
         setUpThread(Thread.currentThread());
 
         RskContext ctx = null;
         try {
             ctx = new RskContext(args);
-
+            if (ctx.isVersionOrHelpRequested()) {
+                System.exit(0);
```

Comment:
what about simply doing `return;` here? I believe `exit()` would prevent the rsk context from proper closure (needs to be checked). Same for `StartBootstrap`

Reply:
yep, makes sense